### PR TITLE
Fix #83 | Add a sample GitHub Workflow section to docs.

### LINKED ISSUE #83: Feature: GitHub Action
A GitHub action to simplify usage in CI would be great.

Just did a quick test with the following:
```
  api-deprecations:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v1

      - name: kubepug
        run: |
          mkdir -p ~/bin
          curl -sSL https://github.com/rikatz/kubepug/releases/download/v1.1.0/kubepug_linux_amd64.tar.gz | tar xvfz - --overwrite -C ~/bin/
          curl -sSL https://get.helm.sh/helm-v3.2.4-linux-amd64.tar.gz | tar xvfz - -C ~/bin/ --wildcards --strip 1 '*/helm'
          
          find charts -mindepth 1 -maxdepth 1 -type d | xargs -t -n1 -I% /bin/bash -c '~/bin/helm template % | ~/bin/kubepug --error-on-deprecated --error-on-deleted --k8s-version v1.16.0 --input-file /dev/stdin'
```

I probably should provide the kubernetes version to helm as well.

### PR DESCRIPTION
**Pull request by ig0rsky**
Fixes #83, unless I understood the intention of the issue creator wrong. :)

### CODE REVIEW

**Comment by ig0rsky** [comment]
Nevermind, I guess they mean that there should be repository for a custom Github Action :D
I misunderstood their intention @rikatz

File: README.md [hunk 1]
```diff
@@ -110,6 +110,33 @@ Steps to follow:
...
+ api-deprecations-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check-out repo
+        uses: actions/checkout@v2
+
+      - name: Install Helm and Kubepug binaries
+        run: |
+          mkdir -p ~/bin
+          curl -sSL https://github.com/rikatz/kubepug/releases/download/${KUBE_VERSION}/kubepug_linux_amd64.tar.gz | tar xvfz - --overwrite -C ~/bin/
```
Comment (rikatz) [inline README.md:133]: ${KUBE_VERSION} should be ${KUBEPUG_VERSION}, right?

**Comment by ig0rsky** [comment]
I mean we could make a custom Github Action and upload that to the marketplace, so that ppl can directly use the tool in their Github Workflow, but I guess this also is enough for others to write their workflows.

**Comment by rikatz** [comment]
Great :) I’ll take a look into this tomorrow. I’m not sure how to write GH Action and put in Marketplace (I know how to use them :P) but that’s a nice idea.

**Comment by rikatz** [comment]
@ig0rsky sorry for being late into this, I'm with some parallel activities consuming all my spare time. Will take a look into the PR ASAP :)

File: README.md [hunk 2]
```diff
@@ -110,6 +110,33 @@ Steps to follow:
...
+ api-deprecations-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check-out repo
+        uses: actions/checkout@v2
+
+      - name: Install Helm and Kubepug binaries
+        run: |
+          mkdir -p ~/bin
+          curl -sSL https://github.com/rikatz/kubepug/releases/download/${KUBEPUG_VERSION}/kubepug_linux_amd64.tar.gz | tar xvfz - --overwrite -C ~/bin/
```
Comment (rikatz) [inline README.md:133]: What about using the latest version everytime? https://github.com/rikatz/kubepug/releases/latest/download/kubepug_linux_amd64.tar.gz

**Comment by rikatz** [comment]
@ig0rsky Thank you very much for the contribution!

I've left another review, about using the latest version instead of a pinned one.

It works like a charm, I've made some tests here: https://github.com/rikatz/pug-tests/runs/1018853600?check_suite_focus=true

I would create an entire section about GH Actions, as in my example I'm not a Helm user but still could use it with some small modifications :)

Again, thank you very much again

**Comment by rikatz** [comment]
Oh and I wont take as much time again to review, was on a rush this last month in the job. So don't worry, this will get merged fast now :)

File: README.md (hunk 2 above)
Comment (ig0rsky) [reply README.md]: done

File: README.md [hunk 3]
```diff
@@ -110,6 +110,32 @@ Steps to follow:
...
+ api-deprecations-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check-out repo
+        uses: actions/checkout@v2
+
+      - name: Install Helm and Kubepug binaries
+        run: |
+          mkdir -p ~/bin
+          curl -sSL https://github.com/rikatz/kubepug/latest/download/kubepug_linux_amd64.tar.gz | tar xvfz - --overwrite -C ~/bin/
```
Comment (rikatz) [inline README.md:132]: It seems there's a part missing here: should be https://github.com/rikatz/kubepug/releases/latest/download/kubepug_linux_amd64.tar.gz (there's a "releases" between the repo name and the /download

**Comment by rikatz** [comment]
@ig0rsky made just a small comment, it seems that the url to download the binary is missing a part :)

**Comment by rikatz** [comment]
Thinking about merging this and correcting the typo in master :D

**Comment by rikatz** [comment]
@ig0rsky friendly ping :D

**Comment by ig0rsky** [comment]
sry mate, forgot to fix the typo :D

**Comment by rikatz** [comment]
LGTM

Thank you very much.

**Pull request merged**